pyxdf/pyxdf.py

```diff
@@ -195,6 +200,8 @@ def load_xdf(filename,
         load the streams contained in a given XDF file
         >>> streams, fileheader = load_xdf('myrecording.xdf')
     """
+    if verbose is not None:
+        logger.setLevel(logging.INFO if verbose else logging.WARNING)
```

Comment:
I think we should store the current log level and restore it at the end of the function. Maybe we could even write a `verbose` decorator for that purpose. This decorator would do all the context management and add a `verbose` argument to the function it wraps.

This is also my second comment - even if you do not want to implement that as a decorator, I think the `verbose` argument should be the last parameter of `load_xdf`.

Reply:
Or in `**kwargs`

Reply:
Yep.

Reply:
Example for decorator: https://www.nmr.mgh.harvard.edu/mne/dev/generated/mne.verbose.html

Reply:
Nope. I don't want to handle kwargs unless we're passing them on to something. There's no need for that here.

We also don't need a decorator for something that appears in exactly one function.

Please, less over-engineering.